src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -479,14 +478,51 @@ public override bool Equals(object obj)
     public override int GetHashCode()
     {
         var hashCode = -284411267;
-        hashCode = hashCode * -1521134295 + base.GetHashCode();
-        hashCode = hashCode * -1521134295 + j.GetHashCode();
+        hashCode = hashCode ^ base.GetHashCode();
+        hashCode = (hashCode << 5) | (hashCode >> 27);
```

Comment:
❓ Did you mean for this to be a rotation? If so, the right shift needs to be unsigned.

Reply:
good question.  let me ask the original hash people about this.

Reply:
Ok.  Switched to unsigned right shift.  Awesome catch!